src/tiledimage.js

```diff
@@ -1958,39 +1958,39 @@ function compareTiles( previousBest, tile ) {
  * Defines the value for subpixel rounding to fallback to in case of missing or
  * invalid value.
  */
-var DEFAULT_SUBPIXEL_ROUNDING_RULE = 'NEVER';
+var DEFAULT_SUBPIXEL_ROUNDING_RULE = $.SUBPIXEL_ROUNDING_OCCURRENCES.NEVER;
 
 /**
  * @private
  * @inner
- * Determines whether the subpixel rounding enum value is 'ALWAYS' or not.
+ * Determines whether the subpixel rounding enum value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.ALWAYS} or not.
  *
- * @param {string} value - The subpixel rounding enum value to check, case sensitive.
- * @returns {Boolean} True if input value is 'ALWAYS', false otherwise.
+ * @param {SUBPIXEL_ROUNDING_OCCURRENCES} value - The subpixel rounding enum value to check.
+ * @returns {Boolean} True if input value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.ALWAYS}, false otherwise.
  */
 function isSubPixelRoundingRuleAlways(value) {
-    return value === 'ALWAYS';
+    return value === $.SUBPIXEL_ROUNDING_OCCURRENCES.ALWAYS;
 }
 
 /**
  * @private
  * @inner
- * Determines whether the subpixel rounding enum value is 'ONLY_AT_REST' or not.
+ * Determines whether the subpixel rounding enum value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.ONLY_AT_REST} or not.
  *
- * @param {string} value - The subpixel rounding enum value to check, case sensitive.
- * @returns {Boolean} True if input value is 'ONLY_AT_REST', false otherwise.
+ * @param {SUBPIXEL_ROUNDING_OCCURRENCES} value - The subpixel rounding enum value to check.
+ * @returns {Boolean} True if input value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.ONLY_AT_REST}, false otherwise.
  */
  function isSubPixelRoundingRuleOnlyAtRest(value) {
-    return value === 'ONLY_AT_REST';
+    return value === $.SUBPIXEL_ROUNDING_OCCURRENCES.ONLY_AT_REST;
 }
 
 /**
  * @private
  * @inner
- * Determines whether the subpixel rounding enum value is 'NEVER' or not.
+ * Determines whether the subpixel rounding enum value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.NEVER} or not.
  *
- * @param {string} value - The subpixel rounding enum value to check, case sensitive.
- * @returns {Boolean} True if input value is 'NEVER', false otherwise.
+ * @param {SUBPIXEL_ROUNDING_OCCURRENCES} value - The subpixel rounding enum value to check.
+ * @returns {Boolean} True if input value is {@link SUBPIXEL_ROUNDING_OCCURRENCES.NEVER}, false otherwise.
  */
  function isSubPixelRoundingRuleNever(value) {
```

Comment:
Now that we're using numbers instead of strings I don't think the `isSubPixelRoundingRule...` methods are necessary anymore, and they add code bulk and (a tiny amount of) processor overhead. Let's lose them. 

Keeping determineSubPixelRoundingRule, normalizeSubPixelRoundingRule, and isSubPixelRoundingRuleUnknown seems reasonable though.